templates/containers/chiselled/index.html

```diff
@@ -0,0 +1,321 @@
+    </div>
+    <div class="col">
+      <p>Chiselled Ubuntu images are fully supported by Canonical, on the same terms as classic minimal Ubuntu images:
+      </p>
+      <hr class="p-rule is-muted" />
+      <ul class="p-list--divided">
+        <li class="p-list__item has-bullet">5-year free bug fixing and security patching for the main libraries</li>
+        <li class="p-list__item has-bullet">10-year security patching for Ubuntu Pro customers, on all Ubuntu packages
+        </li>
+        <li class="p-list__item has-bullet">24/7 phone and ticket customer support</li>
+      </ul>
+      <a href="/security/docker-images">Read more about our support commitments&nbsp;&rsaquo;</a>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width" />
+
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-12">
+      <p class="p-heading--2">
+        <a href="/blog/craft-custom-chiselled-ubuntu-distroless">Get started with chiselled Ubuntu containers
+          today&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Further reading</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-2 col-medium-3">
+          <h3 class="p-text--heading5 p-text--small-caps">Documentation</h3>
+        </div>
+        <div class="col-4 col-medium-3">
+          <ul class="p-list--divided">
+            <li class="p-list__item">
+              <a href="https://canonical-rockcraft.readthedocs-hosted.com/en/latest/explanation/chisel/">Using chisel in
+                Rockcraft</a>
+            </li>
+            <li class="p-list__item">
+              <a
+                href="https://canonical-public-cloud.readthedocs-hosted.com/en/latest/oci/oci-how-to/create-chiselled-ubuntu-image.html">Create
+                a chiselled Ubuntu base image for C/C++, Go and Rust applications</a>
+            </li>
+            <li class="p-list__item">
+              <a href="https://canonical-rockcraft.readthedocs-hosted.com/en/latest/tutorials/chisel/">Install packages
+                slices into a ROCK</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <hr class="is-muted" />
+      <div class="row">
+        <div class="col-2 col-medium-3">
+          <h3 class="p-text--heading5 p-text--small-caps">Docker Hub</h3>
+        </div>
+        <div class="col-4 col-medium-3">
+          <ul class="p-list--divided">
+            <li class="p-list__item">
+              <a href="https://hub.docker.com/u/ubuntu">Ubuntu on Docker Hub</a>
+            </li>
+            <li class="p-list__item">
+              <a href="http://ubuntu/dotnet-runtime">Chiselled Ubuntu for Chiselled .NET</a>
+            </li>
+            <li class="p-list__item">
+              <a href="http://ubuntu/dotnet-deps">Chiselled Ubuntu for Chiselled .NET Deps</a>
```

Comment:
these two links don't work

Reply:
my bad, I just copied them from the copy doc without even checking. Requested the correct ones.